### PR TITLE
Add Caesar cipher algorithm implementation

### PR DESCRIPTION
**Pull request by CuriousCorrelation**
- Add Caesar cipher implementation
- Improved docs according to [#8 ](https://github.com/alexfertel/rust-algorithms/issues/8#issue-938277347)

### CODE REVIEW

File: src/ciphers/mod.rs [hunk 1]
```diff
@@ -1,3 +1,5 @@
 mod transposition;
+mod caesar;
```
Comment (alexfertel) [inline src/ciphers/mod.rs]:
```suggestion
mod caesar;
mod transposition;
```

File: src/ciphers/mod.rs [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
 
 pub use self::transposition::transposition;
+pub use self::caesar::caesar;
```
Comment (alexfertel) [inline src/ciphers/mod.rs]:
```suggestion
pub use self::caesar::caesar;
pub use self::transposition::transposition;
```

File: src/ciphers/caesar.rs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            } else {
+                c
+            }
+        })
+        .collect()
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
```
Comment (alexfertel) [inline src/ciphers/caesar.rs]:
```suggestion
    use super::caesar;
```

**Review by alexfertel — CHANGES_REQUESTED** [review]
Hi! Thank you very much for the very clean and concise PR! :heart:

I left some comments because `rustfmt` is complaining 🙃

**Comment by CuriousCorrelation** [comment]
Just pushed the requested changes, so it should be fixed now! Let me know if I should make a new pull request.

**Comment by alexfertel** [comment]
Thank you very much for your contribution! I actually had forgotten about updating the README. I should add somewhere a reminder to update it.

**Pull request merged**